pysat/instruments/pysat_testadd3.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+"""
+Produces fake instrument data for testing.
+Adapted from existing pysat testing instrument, but changes the data in 
+dummy1 to ascending integers from 10 and deletes dummy2-4
+"""
+
+import pandas as pds
+import numpy as np
+import pysat
+
+# pysat required parameters
+platform = 'pysat'
+name = 'testadd1'
+# dictionary of data 'tags' and corresponding description
+tags = {'':'Regular testing data set'}
+# dictionary of satellite IDs, list of corresponding tags
+sat_ids = {'':['']}
+test_dates = {'':{'':pysat.datetime(2009,1,1)}}
+
+meta = pysat.Meta()
+meta['uts'] = {'units':'s', 'long_name':'Universal Time', 'custom':False}
+meta['mlt'] = {'units':'hours', 'long_name':'Magnetic Local Time'}
+meta['slt'] = {'units':'hours', 'long_name':'Solar Local Time'}
+
+        
+def init(self):
+    self.new_thing=True        
+                
+def load(fnames, tag=None, sat_id=None, sim_multi_file_right=False,
+        sim_multi_file_left=False, root_date = None):
+    # create an artifical satellite data set
+    parts = fnames[0].split('/')
+    yr = int('20'+parts[-1][0:2])
+    month = int(parts[-3])
+    day = int(parts[-2])
+
+    date = pysat.datetime(yr, month, day)
+    if sim_multi_file_right:
+        root_date = root_date or pysat.datetime(2009,1,1,12)
+        data_date = date+pds.DateOffset(hours=12)
+    elif sim_multi_file_left:
+        root_date = root_date or pysat.datetime(2008,12,31,12)
+        data_date = date-pds.DateOffset(hours=12)
+    else:
+        root_date = root_date or pysat.datetime(2009,1,1)
+        data_date = date
+    num = 86400 if tag is '' else int(tag)
+    num_array = np.arange(num)
+    uts = num_array
+    data = pysat.DataFrame(uts, columns=['uts'])
+
+    # need to create simple orbits here. Have start of first orbit 
+    # at 2009,1, 0 UT. 14.84 orbits per day	
+    time_delta = date  - root_date
+    uts_root = np.mod(time_delta.total_seconds(), 5820)
+    mlt = np.mod(uts_root+num_array, 5820)*(24./5820.)
+    data['mlt'] = mlt
+    
+    # fake orbit numbermedC1 = resultsC['dummy1']['median']
+    fake_delta = date  - pysat.datetime(2008,1,1) 
+    fake_uts_root = fake_delta.total_seconds()
+
+    data['orbit_num'] = ((fake_uts_root+num_array)/5820.).astype(int)
+    
+    # create a fake longitude, resets every 6240 seconds
+    # sat moves at 360/5820 deg/s, Earth rotates at 360/86400, takes extra time 
+    # to go around full longitude
+    long_uts_root = np.mod(time_delta.total_seconds(), 6240)
+    longitude = np.mod(long_uts_root+num_array, 6240)*(360./6240.)
+    data['longitude'] = longitude
+
+    # create latitude area for testing polar orbits
+    latitude = 90.*np.cos(np.mod(uts_root+num_array, 5820)*(2.*np.pi/5820.)) 
+    data['latitude'] = latitude
+    
+    # do slt, 20 second offset from mlt
+    uts_root = np.mod(time_delta.total_seconds()+20, 5820)
+    data['slt'] = np.mod(uts_root+num_array, 5820)*(24./5820.)
+    
```

Comment:
This conversion has appeared a few times.  Would it be worth having a little function for it?